az-core/src/main/java/azkaban/Constants.java

```diff
@@ -89,6 +89,8 @@
   // The flow exec id for a flow trigger instance unable to trigger a flow yet
   public static final int FAILED_EXEC_ID = -2;
 
+  // Name of the file which keeps project directory size
+  public static final String PROJECT_DIR_SIZE_FILE_NAME = "_azkaban_project_dir_size_in_bytes_";
```

Comment:
Make it "___*" three underscores to make it even less likely to have a collision?  I assume it will be hard to change the file name once this change is deployed...

Reply:
updated, thanks